app/bundles/CoreBundle/Resources/views/GlobalSearch/globalsearch.html.twig

```diff
@@ -1,36 +1,46 @@
 <li class="dropdown dropdown-custom" id="globalSearchDropdown">
-    <div class="dropdown-menu">
-        <div class="panel panel-default">
-            <div class="panel-heading">
-                <div class="panel-title">
-                    <h6 class="fw-sb">{% trans %}mautic.core.search.results{% endtrans %}</h6>
-                </div>
-            </div>
-            <div class="pt-0 pb-xs pl-0 pr-0">
-                <div class="scroll-content slimscroll" style="height:250px;" id="globalSearchResults">
-                    {{ include('@MauticCore/GlobalSearch/results.html.twig', {results: results}) }}
-                </div>
-            </div>
-        </div>
-    </div>
+	<div class="dropdown-menu">
+
+	</div>
```

Comment:
Do you know if we need this? If so, can we add some comments to the empty div to clarify the purpose?

Reply:
Definitely forgot to remove